File: R/Statistic/II - binom.r
# evento independetes
# mesmas chances de probabilidade por resultado

# jogar uma moeda 5 vezes e cair cara 3 vezes
# probabilidade 50%
dbinom(3, 5, .5)

# 4 sinais de 4 tempos
# probabilidade de 0, 1, 2, 3, 4 abertos
dbinom(0, 4, 0.25) # 0
dbinom(1, 4, 0.25) # 1
dbinom(2, 4, 0.25) # 2
dbinom(3, 4, 0.25) # 3
dbinom(4, 4, 0.25) # 4

# semaforo de 2 tempos
dbinom(0, 4, 0.5) # 0
dbinom(1, 4, 0.5) # 1
dbinom(2, 4, 0.5) # 2
dbinom(3, 4, 0.5) # 3
dbinom(4, 4, 0.5) # 4

# probabilidade acumulativa
pbinom(4, 4, .25) # only 1

#### ---

# prova 12 questões
# chutar todas acertar 7
# quatro alternativas 25%
dbinom(7, 12, .25)

# acetar as 12
dbinom(12, 12, .25)

File: Python/Statistic/II - binom.py
from scipy.stats import binom

# evento independetes
# mesmas chances de probabilidade por resultado

# jogar uma moeda 5 vezes e cair cara 3 vezes
# probabilidade 50%
binom.pmf(3, 5, .5)

# 4 sinais de 4 tempos
# probabilidade de 0, 1, 2, 3, 4 abertos
binom.pmf(0, 4, .25)
binom.pmf(1, 4, .25)
binom.pmf(2, 4, .25)
binom.pmf(3, 4, .25)
binom.pmf(4, 4, .25)

# semaforo de 2 tempos
binom.pmf(0, 4, .5)
binom.pmf(1, 4, .5)
binom.pmf(2, 4, .5)
binom.pmf(3, 4, .5)
binom.pmf(4, 4, .5)

# probabilidade acumulativa
binom.cdf(4, 4, .25)

#### ---

# prova 12 questões
# chutar todas acertar 7
# quatro alternativas 25%
binom.pmf(7, 12, .25)

# acetar as 12
binom.pmf(12, 12, .25)

